contents/blog/google-analytics-to-posthog.md

```diff
@@ -0,0 +1,135 @@
+- Adds functionality like custom event capture, group analytics, session replays, A/B tests, and more.
+
+This post will go over the basics you must know to make the transition from Google Analytics to PostHog.
+
+> To get started, [sign up for PostHog](https://app.posthog.com/signup) and get 1 million events and 15,000 session recordings per month, free.
+
+## Initial PostHog setup
+
+If you’ve set up Google Analytics, PostHog’s setup won’t look too different. After [signing up](https://app.posthog.com/signup), get the script snippet from the "web" option in the getting started flow, and paste it into the `<head>` tag of your site. You can put it right next to your Google Analytics code like this:
+
+```html
+<!-- PostHog snippet -->
+<script>
+    !function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.async=!0,p.src=s.api_host+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="capture identify alias people.set people.set_once set_config register register_once unregister opt_out_capturing has_opted_out_capturing opt_in_capturing reset isFeatureEnabled onFeatureFlags getFeatureFlag getFeatureFlagPayload reloadFeatureFlags group updateEarlyAccessFeatureEnrollment getEarlyAccessFeatures getActiveMatchingSurveys getSurveys".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
+    posthog.init('<ph_project_api_key>',{api_host:'<ph_instance_address>'})
+</script>
+
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-QCX3G7KSPC"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', 'G-QCX3G7KSPC');
+</script>
+```
+
+This works wherever code you can add code to your `<head>` tag, for example, site builders like [WordPress](/docs/libraries/wordpress) and [Webflow](/tutorials/webflow). Alternatively, you can add the PostHog snippet through [Google Tag Manager](/docs/libraries/google-tag-manager) or install the [JavaScript library](/docs/libraries/js) in your app. 
+
+When you set up the snippet or JS library, it autocaptures events (similar to GA4’s [enhanced measurement](https://support.google.com/analytics/answer/9216061)) and they start showing up in your PostHog instance. Pageviews, clicks, inputs, and sessions are all captured without needing to create "properties," "streams," or instrument events yourself.
+
+## Reports and views (aka insights and dashboards)
```

Comment:
I think it's confusing to have Google's names and ours in the title. I think we go with our own and add a boxout / section / table as a summary. Could be something like this:

| **GA Name** | **PostHog Equivalent** | **Description**                                                                                                     |
|-------------|------------------------|---------------------------------------------------------------------------------------------------------------------|
| Report      | Insight                | There x number of insight types in PostHog etc.                                                                     |
| View        | Dashboard              | Dashboards are made out of insights. You can add as many as you like and apply customer filters to whole dashboards |
| etc         | etc                    | etc                                                                                                                 |

Reply:
Added 😄